examples/src/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: MIT
```

Comment:
This file should be up one level in the `examples` directory, not in the `example/src` directory.